### ISSUE TITLE
Add .iml files to. gitignore

### ISSUE THREAD

**Issue by swooboo** [NYRDS/pixel-dungeon-remix#29]
We already discussed this here - https://github.com/NYRDS/pixel-dungeon-remix/pull/28 . (My suggestion is 👍-ed by @Mikhael-Danilov)

To recap:
1. https://github.com/NYRDS/pixel-dungeon-remix/blob/master/PixelDungeon/PixelDungeon.iml is generated by Gradle.
2. Each build that's different from the one before may change this file.
3. This means that this file shouldn't be part of VCS, since its changes are not directly developer-driven.
4. Stack Overflow sheds some light, and suggest not to have `.iml` files as part of VCS.

The issues / questions related to this:
1. Maybe some developer changes in it should be developer driven.
2. Maybe Gradle / Android Studio need this file to already exist for valid builds.
3. Maybe several other `.iml` files need to be `.gitignore`-ed as well? E.g. https://github.com/swooboo/pixel-dungeon-remix/blob/master/Remixed%20Pixel%20Dungeon.iml ?
4. **EDIT**: of course, adding to `.gitignore` should be accompanied by `git rm` and actually deleting the files so they are not tracked anymore.

Let's discuss this further so we are sure that this change is safe. I'll make a PR if we decide to go through with this.

P.S. It brings me exceptional happiness to participate in this project.

**Comment by Mikhael-Danilov** [comment]
Agreed. We can safely remove *.iml files from vcs

P.S. Thanks!

**Comment by swooboo** [comment]
Great, I'll take this. It doesn't seem trivial though. As of now, it seems that at least we'll have to add the Java version somewhere in the Gradle files as well.

**Comment by Mikhael-Danilov** [comment]
Java version is defined in
 https://github.com/NYRDS/pixel-dungeon-remix/blob/master/PixelDungeon/build.gradle

**Comment by Mikhael-Danilov** [comment]
I always used external (to Android Studio) git (and don't know if "export from vcs" scenario ever worked).

But, complieSdkVersion already defined for each flavor. So I assume it will be safe to set it to any value, but lets set it to 26 - as googlePlay flavor.

**Comment by swooboo** [comment]
1. Now that you say it, I checked and "Import from VCS" doesn't work and produces the same error... I thought I imported it like that as well...
2. Setting it to any of the values {25,26,27} yields an error in the `gradle.build` on one (or more) of the dependencies, saying "This support library should not use a different version (27) than the compileSdkVersion (26)". Nevertheless, it compiles and runs.

Anyways, since the same issue appears in the current production as well, if it's "Imported from VCS", this is not something I introduced by deleting the `.iml` files. So people should just use external Git client, and maybe some day somebody will fix this.

I'm submitting a pull request.
